### DISCUSSION TITLE
HyperHDR Support for Hue gradient strips

### DISCUSSION THREAD

**Discussion by DasPhero** [awawa-dev/HyperHDR#176] [Q&A]
Heya. Some time ago Hue released addressable LED strips namely Hue Gradient (Ambience). As I like to use them for normal lighting aswell I would prefer them over WS2812b Strips.
Is there currently any support (currently or later) to control the individual Zones of those strips or is it treated as a singular led strip when using the Hue Integration in the HyperHDR app?
I'd test this but the strips are to expensive to just wing it.

**Comment by awawa-dev** [comment]
Hi
Didn't test it from the same reason: they are too expensive ;) Seems they've done recently some changes in API to better support them but without a hardware or simulator I can't implement it.
